app/components/initEndpoint/initEndpoint.html

```diff
@@ -53,10 +53,6 @@
               <div class="form-group">
                 <div class="col-sm-12">
                   <span class="small">
-                    <p class="text-muted">
-                      <i class="fa fa-exclamation-triangle orange-icon" aria-hidden="true" style="margin-right: 2px;"></i>
-                      This feature is not yet available for <u>native Docker Windows containers</u>.
-                    </p>
                     <p class="text-primary">
                       Please ensure that you have started the Portainer container with the following Docker flag <code>-v "/var/run/docker.sock:/var/run/docker.sock"</code> in order to connect to the local Docker environment.
```

Comment:
The user should also see some hints how to do this with Windows `<code>-v \\.\pipe\\docker_engine:\\.\pipe\\docker_engine</code>`.